server/apps/immich/src/controllers/album.controller.ts

```diff
@@ -22,4 +23,10 @@ export class AlbumController {
   createAlbum(@GetAuthUser() authUser: AuthUserDto, @Body() dto: CreateAlbumDto) {
     return this.service.create(authUser, dto);
   }
+
+  @Authenticated()
```

Comment:
Can be removed, since `@Authenticated()` is already added on controller level

Reply:
Thanks, missed that copying it over.